configs/axibase.json

```diff
@@ -36,9 +36,9 @@
     "lvl1": ".content h1",
     "lvl2": ".content h2",
     "lvl3": ".content h3",
-    "lvl4": ".content h4",
+    "lvl4": ".content h4, .content table tr td:first-child",
     "lvl5": ".content h5",
-    "text": ".content p, .content li, .content pre",
+    "text": ".content p, .content li, .content pre, .content table, .content table td:nth-child(2)",
```

Comment:
Please add the `<p/>` since this selector will also index the noise introduces by **Example:....**

Reply:
@s-pace 
It's hard to implement, our docs are generated from markdown source by `vuepress`. We understand and accept the noise. Probably, in the future, we will try to restructure our docs, but it requires too much time. But know it is ok, that our examples are indexed too. I hope that it doesn't increase index complexity too much.

Reply:
Makes sense. If you are aware and fine with it let's go :)

Reply:
Please note that pages with regular table such as https://axibase.com/docs/atsd/changelogs/2017_feb/ these pages introduce a lot of noise too